src/helm/proxy/clients/human_task_client.py

```diff
@@ -0,0 +1,99 @@
+from abc import ABC, abstractmethod
+
+import random
+from helm.common.hierarchical_logger import hlog
+from helm.common.cache import Cache, CacheConfig
+from helm.common.human_task_request import (
+    HumanQuestionTemplate,
+    HumanTaskRequest,
+    HumanTaskRequestResult,
+    HumanTaskTemplate,
+)
+from cattrs import unstructure
+import threading
+
+from typing import Dict, List
+import surge
+from surge import questions as surge_questions
+
+
+class HumanTaskClient(ABC):
+    @abstractmethod
+    def get_human_task(self, request: HumanTaskRequest) -> HumanTaskRequestResult:
+        pass
+
+
+class RandomHumanTaskClient(HumanTaskClient):
```

Comment:
Not sure we should be calling these human task...since clearly we can have non-human implementations.  In some sense, the scenarios are also human tasks - just that the adapter uses a model (but that could also call out to surge/mturk to get human-level performance).  I kind of think we should try to unify/generalize a bit.

Reply:
How about one of the following: `survey`, `poll`, `questionnaire`, `opinion`?

Other names would look like `SurveyClient`, `SurveyResponses`, `run_survey_evaluation` (or maybe just `run_survey`)?

cc @teetone any opinion here?

Reply:
Notes: questions need not be simple, could be free-form response (so poll isn't appropriate).
Survey kind of suggests that we care about the distribution of humans - not sure if that's really the case here.